doc/source/10min.rst

```diff
@@ -282,7 +282,14 @@ Using a single column's values to select data.
 
    df[df.A > 0]
 
-A ``where`` operation for getting.
+A ``where`` is an attribute of the DataFrame class which helps in getting the results
+based upon the conditional statement that was passed as an argument.
```

Comment:
*"this is implemented by where internally"* @jreback I don't think this is needed in the 10min section, which we should keep as a simple intro. But it can maybe be added in the boolean indexing section in the indexing docs